index.js

```diff
@@ -27,9 +27,9 @@ function getRaceTemplate(raceData) {
   return `
     <div>
       <div>Race Name: ${raceData.raceName}</div> 
-      <ul class="list-group">${raceData.Results.map((result) =>
-        getDriverTemplate(result)
-      )}</ul>
+      <ul class="list-group">
+        ${raceData.Results.map((result) => getDriverTemplate(result)).join("")}
```

Comment:
I just added `.join("")` on the array, to join our driver templates with empty string instead of comma (which is the default)